### ISSUE TITLE
[Bug] DatabaseUtils: java.lang.IllegalArgumentException: bad base-64

### ISSUE THREAD

**Issue by timdriever** [AzureAD/microsoft-authentication-library-for-dotnet#2554]
**Logs and Network traces**
Via the Authenticator app I sent feedback with logging, incident id: RH9JA8G3.

```
04-16 16:15:13.868 11565 11790 E DatabaseUtils: Writing exception to parcel
04-16 16:15:13.868 11565 11790 E DatabaseUtils: java.lang.IllegalArgumentException: bad base-64
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.util.Base64.decode(Base64.java:162)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.util.Base64.decode(Base64.java:137)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.util.Base64.decode(Base64.java:119)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at com.microsoft.identity.client.BrokerContentProvider.query(BrokerContentProvider.java:126)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.content.ContentProvider.query(ContentProvider.java:1235)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.content.ContentProvider.query(ContentProvider.java:1328)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.content.ContentProvider$Transport.query(ContentProvider.java:267)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:104)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.os.Binder.execTransactInternal(Binder.java:1056)
04-16 16:15:13.868 11565 11790 E DatabaseUtils: 	at android.os.Binder.execTransact(Binder.java:1029)
04-16 16:15:13.869 10514 11474 I mono-stdout: MSAL: Error (False) MSAL 4.29.0.0 MSAL.Xamarin.Android 29 [04/16/2021 14:15:13] [Android broker] Failed to get accounts from the broker.  Sender=AzureAdAuthTokenProvider
 ```

**Which Version of MSAL are you using ?**
Msal 4.29.0

**Platform**
Xamarin.Android

**What authentication flow has the issue?**
* Desktop / Mobile
    * [x] Interactive
    * [ ] Integrated Windows Auth
    * [ ] Username Password
    * [ ] Device code flow (browserless)
* Web App 
    * [ ] Authorization code
    * [ ] OBO
* Daemon App 
    * [ ] Service to Service calls

**Is this a new or existing app?**
The app is in production (MSAL 4.24.0), and I have upgraded to a new version of MSAL (4.29.0)

**Repro**
We have several mobile apps that use Authenticator as a broker. This happens consistently while logging in to one of our apps.

**Comment by jennyf19** [comment]
Thanks @timdriever. Are you able to send us MSAL logs as well? Getting logs which match one of the broker incident IDs would be ideal, but if that's not possible, logs from a similarly failed case would be helpful. thank you.

In the incident, it looks like msal v4.24 was working, but updating to 4.29 has issues. Can you revert back to 4.24 until we have investigated the issue?

**Comment by jennyf19** [comment]
@timdriever feel free to email the logs to jeferrie@microsoft.com, and include the issue title in the subject. thanks.

**Comment by timdriever** [comment]
> Thanks @timdriever. Are you able to send us MSAL logs as well? Getting logs which match one of the broker incident IDs would be ideal, but if that's not possible, logs from a similarly failed case would be helpful. thank you.
> 
> In the incident, it looks like msal v4.24 was working, but updating to 4.29 has issues. Can you revert back to 4.24 until we have investigated the issue?

Thanks! I have sent logging to you. We are not live with 4.29.0 yet, it is still on the test environment. We do hope to be able to use version 29 soon because of the improvements.

**Comment by jennyf19** [comment]
what's your "live" date for 4.29? @timdriever

**Comment by bgavrilMS** [comment]
@trwalke - seems like a Content Provider issue

**Comment by timdriever** [comment]
> what's your "live" date for 4.29? @timdriever

We do not have a specific date. As soon as it works again, we would like to release it.

**Comment by pmaytak** [comment]
This is included in [MSAL 4.30.1 release](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/releases/tag/4.30.1).

cc: @timdriever
